coderd/templateversions_test.go

```diff
@@ -1515,3 +1515,117 @@ func TestTemplateVersionParameters_Order(t *testing.T) {
 	require.Equal(t, secondParameterName, templateRichParameters[3].Name)
 	require.Equal(t, thirdParameterName, templateRichParameters[4].Name)
 }
+
+func TestTemplateArchiveVersions(t *testing.T) {
+	t.Parallel()
+
+	client := coderdtest.New(t, &coderdtest.Options{IncludeProvisionerDaemon: true})
+	user := coderdtest.CreateFirstUser(t, client)
```

Comment:
```suggestion
	owner := coderdtest.CreateFirstUser(t, client)
```

Also, can you create a `TemplateAdmin` user and test with that instead.